source/reference/icon.txt

```diff
@@ -46,7 +46,7 @@ This role uses the
 `FontAwesome 5 Solid <https://fontawesome.com/icons?d=gallery&s=solid&m=free>`__
 icon font.
 
-.. icon-fa5-help::
+.. .. icon-fa5-help::
```

Comment:
I don't think these are rendering how they are supposed to (applies to all "icon-help" references on this page)

Reply:
I purposely commented them out because it was erroring out. I was going to either let whoever reviews the entire branch make a call on whether to keep it or what the fix is.